docs/contributing/ui_guidelines.md

```diff
@@ -0,0 +1,47 @@
+# UI Guidelines
```

Comment:
I think that 'UI guidelines' makes sense, we want to be a bit more aspirational here and help our developers understand that when writing UI it is not just the languages but the overall goals.

Also front-end/back-end is hard to treat as a clean split, for example HTML can be written in Django templates and in React.

I think we can get this in in roughly the current state and new a new revision that helps make this easier to run with for developers in the future (maybe a new issue).

Reply:
Yes, I think this is also not ideal as-is because the HTML/CSS/JS guidelines have a very narrow focus. My hope is to have a much more cross-functional page eventually, that would cover all of UX, visual design, front-end.